### DISCUSSION TITLE
Stop Auto Creation of Anime and Realistic.bat

### DISCUSSION THREAD

**Discussion by AliciaWindsor** [lllyasviel/Fooocus#3722] [Q&A] [answered]
Just curious, is there a way to stop this garbage from being recreated each time?

**Comment by mashb1t** [comment, marked as answer]
Yes, you can comment or remove line https://github.com/lllyasviel/Fooocus/blob/d7439b2d6004d50a0fda19108603a8d1941a185e/launch.py#L76
